src/include/mpir_thread.h

```diff
@@ -43,15 +43,15 @@ extern MPIR_Thread_info_t MPIR_ThreadInfo;
 typedef struct {
     /* error string storage for MPIR_Strerror */
     char strerrbuf[MPIR_STRERROR_BUF_SIZE];
-} MPIR_Per_thread_t;
+} MPIR_Thread_tls_obj_t;
 
 #if defined(MPICH_IS_THREADED) && defined(MPL_TLS)
-extern MPL_TLS MPIR_Per_thread_t MPIR_Per_thread;
+extern MPL_TLS MPIR_Thread_tls_obj_t MPIR_Thread_tls_obj;
 #else
-extern MPIR_Per_thread_t MPIR_Per_thread;
+extern MPIR_Thread_tls_obj_t MPIR_Thread_tls_obj;
```

Comment:
Are we concerned of the case when `MPICH_IS_THREADED` but no compiler support for TLS?

Reply:
Yes, not all threading libraries have compiler support, so we use the runtime interface for them.  But that code already exists in MPICH.  I'm not adding or removing it in this PR.